modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -0,0 +1,320 @@
++
+Alternatively, you can use *Auto* mode to automatically create the Operator roles and OIDC provider.
++
+[NOTE]
+====
+To enable *Auto* mode, the {cluster-manager} role must have administrator capabilities.
+====
+
+. Optional: Specify a *Custom operator roles prefix* for your cluster-specific Operator IAM roles. 
++
+[NOTE]
+====
+By default, the cluster-specific Operator role names are prefixed with the cluster name and random 4-digit hash. You can optionally specify a custom prefix to replace `<cluster_name>-<hash>` in the role names. The prefix is applied when you create the cluster-specific Operator IAM roles. For information about the prefix, see _Defining a custom Operator IAM role prefix_.
+====
+
+. Select *Next*.
+
+. On the *Cluster update strategy* page, configure your update preferences:
+.. Choose a cluster update method:
+** Select *Individual updates* if you want to schedule each update individually. This is the default option.
+** Select *Recurring updates* to update your cluster on your preferred day and start time, when updates are available.
++
+[NOTE]
+====
+You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
+====
++
+.. If you opted for recurring updates, select a preferred day of the week and upgrade start time in UTC from the drop-down menus.
+.. Optional: You can set a grace period for *Node draining* during cluster upgrades. A *1 hour* grace period is set by default.
+.. Click *Next*.
++
+[NOTE]
+====
+In the event of critical security concerns that significantly impact the security or stability of a cluster, Red Hat Site Reliability Engineering (SRE) might schedule automatic updates to the latest z-stream version that is not impacted. The updates are applied within 48 hours after customer notifications are provided. For a description of the critical impact security rating, see link:https://access.redhat.com/security/updates/classification[Understanding Red Hat security ratings].
+====
+
+. Review the summary of your selections and click *Create cluster* to start the cluster installation.
+
+. If you opted to use *Manual* mode, create the cluster-specific Operator roles and OIDC provider manually to continue the installation:
+.. Under the *Installing cluster* section on the *Overview* page for your cluster, select *Installation waiting* -> *Review your options*.
+.. In the *Action required to continue the installation* dialog, select either the *AWS CloudFormation*, *AWS CLI*, or *ROSA CLI* tab and manually create the resources:
+** If you opted to use the *AWS CloudFormation* method, click the copy button next to the `aws cloudformation` commands and run them in the CLI.
+** If you opted to use the *AWS CLI* method, download the `Download.zip` file and extract the AWS CLI commands and policy files. Then, run the commands in the CLI.
```

Comment:
We might call out specifically that the commands need to be run in the directory with the policy files from the zip.

Reply:
@spadgett thank you for your review and for highlighting this. I have subsequently added a note under this step to call this out.